test/puppetlabs/pcp/messaging_test.clj

```diff
@@ -57,8 +57,7 @@
   {:server      "wss://localhost:8143/pcp/"
    :cert        (format "test-resources/ssl/certs/%s.example.com.pem" cn)
    :private-key (format "test-resources/ssl/private_keys/%s.example.com.pem" cn)
-   :cacert      "test-resources/ssl/certs/ca.pem"
-   :type        "demo-client"})
```

Comment:
I removed the types because the default case is for them no longer to be supplied. I'm fine with putting one or more of them back, but I don't know that it's something we should be adopting for e.g new tests, or even showing in the readme examples, unless we want to actually make it required.